src/NuGet.Clients/NuGet.SolutionRestoreManager/BrokeredServicesUtility.cs

```diff
@@ -11,9 +11,17 @@ namespace NuGet.SolutionRestoreManager
     internal static class BrokeredServicesUtility
     {
         // It is very important that these names and versions are kept the same as the ones in NuGet.VisualStudio.Internal.Contracts.NuGetServices.
-        internal const string NuGetSolutionServiceName = "NuGetSolutionService";
+        internal const string NuGetSolutionServiceName = "Microsoft.VisualStudio.NuGet.SolutionService";
```

Comment:
@nkolev92 Please review as we are changing the name of this service.

Reply:
sure, I'll take a look.